### PR TITLE
test: fix Azure tests after MEN-5386

### PR DESCRIPTION
**Pull request by tranchitella**
Changelog: none

Signed-off-by: Fabio Tranchitella <fabio.tranchitella@northern.tech>

### CODE REVIEW

**Comment by tranchitella** [comment]
Required by https://github.com/mendersoftware/iot-manager/pull/26

File: backend-tests/tests/test_azure.py [hunk 1]
```diff
@@ -117,9 +117,19 @@ def check_integrations(self, user: User, expected_integration: Dict):
...
+            if part.startswith('SharedAccessKey='):
+                part = part[:20]
             assert part in expected_integration["credentials"]["connection_string"]
         # Check that expected properties are a subset of actual integrations
         for part in expected_integration["credentials"]["connection_string"].split(";"):
+            # SharedAccessKey will be masked, with only the first 4 characters visible
+            # and the rest of the string replaced with a place holder. For this reason,
+            # we'll test the first 20 bytes only
+            if part.startswith('SharedAccessKey='):
+                part = part[:24]
```
Comment (alfrunes) [inline backend-tests/tests/test_azure.py:132]:
```suggestion
                part = part[:20]
```

File: backend-tests/tests/test_azure.py [hunk 2]
```diff
@@ -117,9 +117,19 @@ def check_integrations(self, user: User, expected_integration: Dict):
         # Check for equality by parts:
         # Check that actual properties are a subset of expected integrations
         for part in actual.split(";"):
+            # SharedAccessKey will be masked, with only the first 4 characters visible
+            # and the rest of the string replaced with a place holder. For this reason,
+            # we'll test the first 20 bytes only
+            if part.startswith('SharedAccessKey='):
+                part = part[:20]
             assert part in expected_integration["credentials"]["connection_string"]
```
Comment (alfrunes) [inline backend-tests/tests/test_azure.py:125]: Doesn't both sides of the assertion require truncating the access key?

Comment (tranchitella) [reply backend-tests/tests/test_azure.py:125]: no, I don't think so. The portion we take from part should be contained in the string.

Comment (alfrunes) [reply backend-tests/tests/test_azure.py:125]: Right! I thought that the expected value was a list of the connection string key/value pairs.

**Review by alfrunes — APPROVED** [review]

**Pull request merged**